library/psa_crypto.c

```diff
@@ -1805,12 +1994,19 @@ static psa_status_t psa_validate_optional_attributes(
 #if defined(MBEDTLS_RSA_C)
         if( PSA_KEY_TYPE_IS_RSA( slot->attr.type ) )
         {
+            mbedtls_rsa_context rsa;
+            mbedtls_rsa_init( &rsa, MBEDTLS_RSA_PKCS_V15, MBEDTLS_MD_NONE );
+
+            psa_status_t status = psa_load_rsa_representation( slot, &rsa );
+            if( status != PSA_SUCCESS )
+                return status;
             mbedtls_mpi actual, required;
```

Comment:
Keep those declarations at the beginning of the block (line 1998)?

Reply:
Sure, will update momentarily.